IntelTurboBoost@wsmenezes/README.md

```diff
@@ -0,0 +1,128 @@
+![Technology](https://img.shields.io/badge/javascript-powered-green.svg)
+[![License](https://img.shields.io/badge/license-MIT-lightgrey.svg)](https://opensource.org/licenses/MIT)
+
+## Table of contents
+
+<!-- TOC -->  
+  * [Description](#description)
+  * [Disclaimer](#disclaimer)
+  * [Processors Tested](#processors-tested)
+  * [Requirements](#requirements)
+    * [Installing requirements on Linux](#installing-requirements-on-linux)
+  * [Installation / Removal](#installation--removal)
+  * [Screenshot](#screenshot)
+  * [Troubleshooting](#troubleshooting)
+  * [Issues](#issues)
+  * [License](#license)
+  * [Credits](#credits)
+<!-- TOC -->
+
+## Description
+
+**cinnamon-applet-IntelTurboBoost** is a simple systray applet designed for Cinnamon desktop environments.
+
+**Intel Turbo Boost©** is a remarkable technology, yet there are numerous reports on the Internet regarding potential 
+instability, ranging from system crashes to thermal throttling. 
+
+Most of the time these issues are related to the motherboard, not the processor itself. In my case the root cause was a 
+well known issue regarding my motherboard [VRM](https://www.tomshardware.com/reviews/vrm-voltage-regulator-module-definition,5771.html). Budget motherboards...
+
+Until I discovered and fixed my issue I had to disable **Intel Turbo Boost©** at **BIOS** whenever I wanted to have a 
+stable system for critical tasks.
+
+This applet simplifies the process of enabling or disabling **Intel Turbo Boost©** while your system is running, 
+eliminating the need for a full system reboot. If you are struggling with **Intel Turbo Boost©**, here are some 
+scenarios in which you might consider disabling the feature (for the moment at least):
+
++ Banking applications.
++ Avoiding thermal throttling.
++ Achieving extreme power savings.
+
+**Important:** Please be aware that if **Intel Turbo Boost©** is disabled at the BIOS level, 
+this applet **cannot override** those settings. It serves as a convenient in-session toggle 
+for **Intel Turbo Boost©**.
+
+## Disclaimer
+
+This tool may potentially cause damage to your hardware due to its use of reverse-engineered methods involving 
+**Model Specific Registers (MSRs)**. Use it at your own risk.
+
+## Processors Tested
+
+These are the processors the applet was successfully tested:
+
++ **i5-10400F**
++ **i5-7200U**
+
+You are more than welcomed to update this list.
+
+## Requirements
+
+**cinnamon-applet-IntelTurboBoost** requires **msr-tools** package.
+
+### Installing requirements on Linux
+
+Make sure **msr-tools** and **python3-polib** packages are installed before using **cinnamon-applet-IntelTurboBoost**.
+
+On **Ubuntu/Debian** systems, install them with:
+
+~~~bash
+$ sudo apt-get install msr-tools
+$ sudo apt-get install python3-polib
+~~~
+
+On **Redhat/Fedora** systems, install them with:
+
+~~~bash
+$ yum install msr-tools
+$ yum install python3-polib
+~~~
+    
+## Installation / Removal
```

Comment:
Just an FYI, this is handled once it is added to the repository of Spices and available from the Applets dialog in Cinnamon.

Reply:
Yes, thanks for catching that. Do you think it would be better to remove that to avoid any confusion? Please let me know your thoughts. Thanks again for the review.

Reply:
It's your call! As long as users aren't confused. I had to check myself that the applet didn't require manual compilation as that wouldn't be the best user experience. Maybe borrow from the README of other applets and encourage users to install via their system if possible, then provide alternatives if something goes wrong?

Reply:
If you look on your own system, you should see this applet in the system Applets dialog and you can make sure it's working as intended.

Reply:
> It's your call! As long as users aren't confused. I had to check myself that the applet didn't require manual compilation as that wouldn't be the best user experience. Maybe borrow from the README of other applets and encourage users to install via their system if possible, then provide alternatives if something goes wrong?

Got it. I'll do that, replicate some section like you suggested. It's better. I'll request an update.